editor/editor_node.cpp

```diff
@@ -4400,6 +4413,8 @@ void EditorNode::_quick_opened() {
 		const String &res_path = files[i];
 		if (open_scene_dialog || ClassDB::is_parent_class(ResourceLoader::get_resource_type(res_path), "PackedScene")) {
 			open_request(res_path);
+		} else if (ClassDB::is_parent_class(ResourceLoader::get_resource_type(res_path), "ColorPalette")) {
+			palette_file_selected_callback.call_deferred(files[i]);
```

Comment:
```suggestion
			palette_file_selected_callback.call_deferred(res_path);
```
Available convenience used elsewhere